src/Processors/QueryPlan/ReadFromMergeTree.cpp

```diff
@@ -2002,8 +2011,12 @@ void ReadFromMergeTree::initializePipeline(QueryPipelineBuilder & pipeline, cons
                 Context::QualifiedProjectionName{.storage_id = data.getStorageID(), .projection_name = storage_snapshot->projection->name});
     }
 
+    ProfileEvents::increment(ProfileEvents::TotalParts, result.total_parts);
```

Comment:
I am really confused now: Four new metrics are introduced, and by the look of things they all take on the same value. Why would one introduce them in the first place? Can't users simply use `SelectedParts`, for example?

Reply:
EDIT: I see ... the values are different, sorry.